proxy/src/control/discovery.rs

```diff
@@ -275,8 +281,8 @@ where
                             // we may already know of some addresses here, so push
                             // them onto the new watch first
                             match set.addrs {
-                                Exists::Yes(ref mut addrs) => {
-                                    for &addr in addrs.iter() {
+                                Exists::Yes(Cache { ref values, reset_on_next_modification: _ }) => {
```

Comment:
to confirm i understand what this syntax indicates: this is ignoring the field, and you opted not to use `, ..` for the sake of being explicit?

Reply:
Yes, exactly.

Reply:
Is there a reason this uses a larger match like that, instead of just something like:

```rust
match set.addrs {
    Exists::Yes(ref cache) => {
        for &addr in cache.values.iter() {
        // ....
    }
}
```

Reply:
I agree that's clearer in this context. I'll change it.